launcher/screens/Guide/Dashboard/BattlefieldOverview/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Image, StyleSheet, View } from 'react-native';
+import { DimensionState, dimensionState, Text } from '@metacraft/ui';
+import { battlefield } from 'screens/Guide/content';
+import ConceptButton from 'screens/Guide/Dashboard/ConceptButton';
+import { sharedStyle } from 'screens/Guide/shared';
+import { useSnapshot } from 'utils/hook';
+import resources from 'utils/resources';
+
+const styles = StyleSheet.create({
```

Comment:
Please move styles to the end of component. As react native recommend, "moving styles away from the render function, you're making the code easier to understand"